docs/pages/develop/user-interface/fonts.mdx

```diff
@@ -43,9 +44,47 @@ For reference, the following table provides what formats work on which platforms
 
 ## Use a custom font
 
+There are two ways to use a custom font in your project: you can either embed the font in your native project or load it at runtime. The first approach is recommended as it is simpler and more reliable. The second approach is useful if you want to load the font in Expo Go / without creating a new native build that includes it, or load it from a remote URL.
+
+### 1. Embed the font in your native project
```

Comment:
```suggestion
### Embed the font in your native project
```

This seems like either/or approach so let's not use numbers in headers as it causes confusion if the steps are continuous. If my understanding is wrong here, let me know and I will update these sections to use Step component.